assets/css/style.css

```diff
@@ -1,3 +1,276 @@
-.makeHeaderRed {
-  color: red;
+*,
+*::before,
+*::after {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
- [ ] I noticed that the fonts you are using are not imported or added to the as a link in the head tag of the HTML. You need to import them to be able to use them and make them reflect. You can read this [article](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css) or this [article](https://www.pagecloud.com/blog/how-to-add-custom-fonts-to-any-website) to know how to import fonts 👍 💻